docker-compose.yml

```diff
@@ -71,7 +71,7 @@ services:
       - ./psalm.xml:/usr/src/code/psalm.xml
       - ./tests:/usr/src/code/tests
       - ./app:/usr/src/code/app
-      # - ./vendor:/usr/src/code/vendor
+      - ./vendor:/usr/src/code/vendor
```

Comment:
Better keep this as a comment by default.

Reply:
Removed all vendor volumes I added.